packages/components/psammead-episode-list/src/MediaIndicator.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styled from 'styled-components';
+import { string } from 'prop-types';
+import { GEL_SPACING, GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
+import { C_WHITE } from '@bbc/psammead-styles/colours';
+
+const Wrapper = styled.div`
+  display: inline-block;
+  width: ${props => props.size};
+  height: ${props => props.size};
+  margin-left: ${GEL_SPACING};
+  margin-right: ${GEL_SPACING_DBL};
+  vertical-align: top;
+`;
+
+const MediaIndicator = ({ size }) => (
+  <Wrapper
+    aria-hidden="true"
+    size={size}
+    dangerouslySetInnerHTML={{
```

Comment:
This svg should be moved to psammead-assets (eventually - doesn't need to be this PR because the version bumping will be a pain)

Reply:
Why are we dangerously setting inner html?